release-notes/20230704-libreswan-downtime-improve.md

```diff
@@ -0,0 +1 @@
+Reduced Libreswan data path downtime when GW pod restarts.
```

Comment:
What about the following sentence?

Reduced data path downtime with Libreswan cable driver when gateway pod restarts.

Reply:
Also note there is a Markdown failure.